recipes/ohpipeline/all/conandata.yml

```diff
@@ -1,4 +1,13 @@
 sources:
   "1.139.1000":
-    url: "https://github.com/merakiacoustic/ohPipeline/archive/refs/heads/ohMediaPlayer_1.139.1000_meraki1.tar.gz"
-    sha256: "8bc1baaf6a7030ff572e67192f1ed672be531ebb1e36536a66c98ddce7e6425d"
+    source:
+      url: "https://github.com/openhome/ohPipeline/archive/refs/tags/ohMediaPlayer_1.139.1000.tar.gz"
+      sha256: "070eac6d974bbe520d0b292df1ca4a72160a2b607a60dabdaa0971c3ef96e5e4"
+    cmake:
```

Comment:
Out of curiosity: why not use the entire fork?

Reply:
It's preferable to use the original sources, if possible, to make it clear that the library sources themselves are unmodified and that the only change is to the build system. This helps avoid the risk of supply-chain attacks (nothing personal, just a best practice) and makes it easier to check the changes applied by your fork in the case of a version bump in the future.